searx/engines/community_scripts_proxmoxve.py

```
@@ -67,32 +66,74 @@ def _fetch_scripts() -> list[dict[str, t.Any]]:
 
     try:
-        with urllib.request.urlopen(_API_URL, timeout=30) as resp:  # noqa: S310
-            data = json.loads(resp.read().decode())
+        with http.client.HTTPSConnection("community-scripts.github.io", timeout=30) as connection:
+            connection.request("GET", "/ProxmoxVE/api/categories")
+            response = connection.getresponse()
+            if response.status != 200:
+                _logger.warning("Unexpected community scripts API status: %s", response.status)
+                return []
+            data = json.loads(response.read().decode())
     except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)